alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -61,6 +62,7 @@ data class Alert(
     data class Action(
         val title: String,
         val style: Style = Style.DEFAULT,
+        val color: Color? = null,
```

Comment:
Is it color of background, title or message?

Reply:
Or button?

Reply:
Does null color means it's transparent? :)

Reply:
If color it's null it will use default parameter. I will add some doc

Reply:
Or just rename colour to the textColour

Reply:
102db12

Reply:
Which button? All buttons?

Reply:
What if I have destructive dialogue, where danger action has to be red?

Reply:
Ah, I misread, so it's fine

Reply:
On android it works for the single button, but on ios I couldn't find a way to set color for only one action (assign latest color in action list to all actions). 
>Which button? All buttons?

Then you don't pass a color and pass destructive style or pass red color. 
>What if I have destructive dialogue, where danger action has to be red?